tests/machine.test.ts

```diff
@@ -93,7 +100,7 @@ describe(OlmMachine.name, () => {
                 await OlmMachine.initialize(
                     new UserId("@foo:bar.org"),
                     new DeviceId("baz"),
-                    store_name,
+                    undefined,
```

Comment:
The code itself is happy with either, but the typescript definition generated by wasm-bindgen is `static initialize(user_id: UserId, device_id: DeviceId, store_name?: string, store_passphrase?: string): Promise<any>`: in other words, it is an [optional parameter](https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters), so the effective type is `string|undefined`. This makes sense because if you simply omit the parameter (eg, `OlmMachine.initialize(userId, deviceId)`), then `store_name` ends up being set to `undefined`.

the point of it being `undefined` is that